Article/Article.less

```diff
@@ -9,8 +10,10 @@
   background-color: white;
   position: relative;
   height: 50px;
+  transition: height 1s ease;
   overflow: hidden;
   box-shadow: 2px 2px 2px lightgrey;
+  cursor: pointer; // The pointer should communicate to the user that they can click anywhere on the article, not just the small expand/close button.
```

Comment:
very nice using css transitions and the `ease` method to accomplish the transtition you were after